manifests/config/project.pp

```diff
@@ -17,6 +17,7 @@
 #   Rundeck jobs related to a project.
 #
 define rundeck::config::project (
+  Enum['absent', 'present'] $ensure = 'present',
```

Comment:
please add puppet-strings documentation for the new parameter

Reply:
This is already available the unit tests would fail if i didn't run puppet strings

Reply:
ah I was in the wrong line, I was speaking about `$ensure`. That's not documented. And our unit tests don't do anything with the documentation.

Reply:
![image](https://github.com/voxpupuli/puppet-rundeck/assets/25951077/92113346-380b-47cb-99f3-114f1b3c8a1a)
This would fail if a parameter is not documented.
I ran 'bundle exec rake strings:generate:reference' again and no changes so I think you are looking over something or I'm missing something :)

Reply:
As you can see in https://github.com/voxpupuli/puppet-rundeck/blob/master/.puppet-lint.rc#L2 we don't validate if every parameter has documentation. And the header of the file has no documentation for `$ensure`.

Reply:
Ah ok I see what you are trying to say haha, I will update it.

Reply:
Done :)